src/WorkOS.net/Services/_common/Entities/List_Metadata.cs

```diff
@@ -5,7 +5,7 @@
     /// <summary>
     /// Contains pagination options for WorkOS records.
     /// </summary>
-    public class ListMetadata
+    public class List_Metadata
```

Comment:
We should keep the casing here the same, since PascalCase is idiomatic for .NET (and it doesn't need to match the casing of the field in the JSON payload).

```suggestion
    public class ListMetadata
```